src/renderer/src/lib/helpers/other.ts

```diff
@@ -126,10 +146,13 @@ export const fetchFilterLists = async (appstoreClient: AppstoreAppClient, isDev:
 	const allowListsUrl = `https://raw.githubusercontent.com/holochain/appstore-lists/${branch}/${dnaHashBase64}/lists.json`;
 
 	try {
-		const response = await fetch(allowListsUrl);
-		return AppstoreFilterListsSchema.parse(await response.json());
+		const response = await fetchWithTimeout(allowListsUrl, { cache: 'no-cache' });
+		const responseData = await response.json();
+		await localforage.setItem(allowListsUrl, responseData);
+		return AppstoreFilterListsSchema.parse(responseData);
 	} catch (error) {
-		return AppstoreFilterListsSchema.parse({});
+		const data = await localforage.getItem(allowListsUrl);
```

Comment:
it will turn it into

```
{
allowlists: {},
denylist: []
}
```

as defined here

```
export const AppstoreFilterListsSchema = z.object({
	allowlists: z.record(ListName, AppStoreAllowListSchema).default({}),
	denylist: AppStoreDenyListSchema.default([])
});
```

Reply:
Are you saying that the `.parse()` function does this under the hood?
This would mean that in this scenario, although it's unlikely in practice, all apps would have a warning...maybe it would be better to throw an actual error in that case.

Reply:
and display no apps in appstore, instead of with the wrong filtering.

Reply:
While this scenario is indeed very unlikely, the possibility of keeping all apps marked as unverified seems like a reasonable worst-case outcome.

Reply:
But if you still think that the best way to handle this is to show no apps at all, I can do that.

Reply:
fixed